### ISSUE TITLE
managing file ownership re: vagrant ssh/drush

### ISSUE THREAD

**Issue by tannerjfco** [arknoll/drupal-nfs#5]
So drupal-nfs works pretty damn well for me, and I encounter no permissions fun manipulating files from my host. I do, however, run into permission issues using drush via vagrant ssh, which runs as the vagrant user and can't manipulate files the files that are all owned by www-data unless i sudo. Is there a good way to address this?

**Comment by timodwhit** [comment]
Yeah, we have noticed that to, we manage that issue by running things as sudo. 

So when logged in `sudo su`

or run them as sudo. `sudo drush cc all`

**Comment by tannerjfco** [comment]
Would something like ACL and/or umask help with this? I installed the ACL package and gave this a go on the files directory, it helped a migration to process, though it still had some permission errors after trying to set permissions on the files:

```
setfacl -R -m d:u:www-data:rwx,u:www-data:rwx,d:u:vagrant:rwx,u:vagrant:rwx files
```

Error:

```
WD file: The file permissions could not be set on public://pages
```

If we could get the file ownership to be kept as the www-data user, this could probably work. Maybe this and a umask could achieve this, but I'm not overly familiar with umask to be honest.
